### ISSUE TITLE
Cloudflared integration broken due to UmbrelOS login page

### ISSUE THREAD

**Issue by krzwiatrzyk** [getumbrel/umbrel#1805]
TL;DR:

Cloudflare application installed via app store. Portainer app installed via app store, portainer is exposed via 9000 port on Umbrel.

If I will use `<local ip>:9000` or `umbrel.local:9000` to expose Umbrel via Cloudflare Tunnel, I will get:
- https to http redirect error
- redirect to port `2000` for Umbrel authentication.

It can be easily replicated locally, try to login to application using machine IP address, due missing cookie for that address in the browser user will be asked to log in on some random port.

TL;DR:
How can I expose application using Cloudflared now? If I get it right, this will cause errors on any attempt to expose applications via custom public domain

**Comment by highghlow** [comment]
A similar thing happened to transmission: it's rpc port is protected by app_auth, but it's fine because other apps connect to it not by using the external network, but directly inside the docker one. This bypassed app_auth. Based on the screenshots from cloudflate tunnel's appstore page, it also connects to the apps using this method (uses domain names like protoprism_web_1). It the end if cloudflare doesn't work, you can use Tailscale.
